index.js

```diff
@@ -106,7 +106,7 @@ function main(argv, delegate) {
         config.endpoint = 'Meta::health';
         config.traceSample = 0;
     } else {
-        config.traceSample = 1;
+        config.traceSample = 0;
```

Comment:
The last condition blocks is no longer necessary:

```js
        ...
        delegate = delegate || new HealthLogger();
        config.thrift = path.join(__dirname, 'meta.thrift');
        config.endpoint = 'Meta::health';
    }
    config.traceSample = 0;
```

Reply:
Good point.